diff --git a/postgres/contrib/parquet_gsi/meson.build b/postgres/contrib/parquet_gsi/meson.build
new file mode 100644
index 00000000000..954c2d2c50b
--- /dev/null
+++ b/postgres/contrib/parquet_gsi/meson.build
@@ -0,0 +1,11 @@
+parquet_gsi = shared_module('parquet_gsi',
+  files('parquet_gsi.c'),
+  kwargs: contrib_mod_args,
+)
+contrib_targets += parquet_gsi
+
+install_data(
+  'parquet_gsi.control',
+  'parquet_gsi--1.0.sql',
+  kwargs: contrib_data_args,
+)
